app/services/CaptchaService.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { ActivityIndicator, Dimensions, StyleSheet } from 'react-native';
+import WebView from 'react-native-webview';
+
+import Colors from '../constants/colors';
+
+const width = Dimensions.get('window').width;
+/**
+ *
+ * @param {*} onMessage: callback after received response, error of Google captcha or when user cancel
+ * @param {*} siteKey: your site key of Google captcha
+ * @param {*} style: custom style
+ * @param {*} url: base url
+ */
+export const HCaptcha = ({ onMessage, style, uri }) => {
```

Comment:
So at this point it seems like this is a more generic component than a Captcha service - it's essentially a webview wrapper. 

Do we need to pull the captcha logic out into a distinct component if the only unique part is the uri that's being passed?